### PR TITLE
Better docs about redis persistent id

### PR DESCRIPTION
**Pull request by pensiero**
Refer to https://github.com/zendframework/zend-cache/issues/4

### CODE REVIEW

**Comment by Maks3w** [comment]
PR conflicts. please rebase against current master.

**Comment by pensiero** [comment]
@Maks3w done :+1:

File: docs/src/modules/zend.cache.storage.adapter.rst [hunk 1]
```diff
@@ -746,26 +746,26 @@ The Redis Adapter
...
+   |lib_option          |``array``             |``[]``                   | Associative array of redis options were the array key is the option name                     |
+   +--------------------+----------------------+-------------------------+----------------------------------------------------------------------------------------------+
+   |namespace_separator |``string``            |":"                      |A separator for the namespace and prefix                                                      |
+   +--------------------+----------------------+-------------------------+----------------------------------------------------------------------------------------------+
+   |password            |``string``            |""                       |Set password                                                                                  |
+   +--------------------+----------------------+-------------------------+----------------------------------------------------------------------------------------------+
+   |persistent_id       |``string``            |````                     |Set persistent id (*name* of the connection, [read more](http://redis.io/topics/persistence)) |
```
Comment (marc-mabe) [inline docs/src/modules/zend.cache.storage.adapter.rst]: The link is wrong as the link describes the persistence of items stored on the server but this option names persistence of the connection.

Comment (pensiero) [inline docs/src/modules/zend.cache.storage.adapter.rst]: This is exactly the point of the PR. Actually it's specified that persistent_id could be `RDB` or `AOF`, while this is something related to server configuration. So this way user understand that the persistent_id should be only a generic string, while the control of the persistence is done via server (and if they want to know more about this, they can check the docs about Redis Persistence).

Comment (marc-mabe) [inline docs/src/modules/zend.cache.storage.adapter.rst]: Sure this is correct but you shouldn't link a documentation about something very different. Please only note that this `persistent_id` has nothing to do with persistence of the server.

PS: I'm also missing a note that an empty string will result in not using a persistent connection at all.

Comment (pensiero) [inline docs/src/modules/zend.cache.storage.adapter.rst]: Edited, can you check if it's ok?

Comment (marc-mabe) [inline docs/src/modules/zend.cache.storage.adapter.rst]: :+1:

**Comment by pensiero** [comment]
Can we merge it?

**Pull request merged**